src/ts/waterfall/details-overlay/overlay-changes-pub-sub.ts

```diff
@@ -15,6 +16,7 @@ export function subscribeToOverlayChanges(fn: OverlayChangeSubscriber) {
 
 // no need for unsubscribe in the moment
 
-export function publishToOverlayChanges(change: OverlayChangeEvent) {
-  subscribers.forEach((fn) => fn(change));
-}
+export function publishToOverlayChanges(change: OverlayChangeEvent, overlayHolder: SVGGElement) {
+  let eventChartBaseEl = getBaseEl(overlayHolder);
```

Comment:
The problem is that at the component are not attached to any parent at subscription time, this is why I have solved it this way. However `eventChartBaseEl` is not used by any subscriber - and can be removed.

I've timed the traversal for 58 subscribers and it was less than a millisecond.